src/Clap-Core/ClapMismatch.class.st

```diff
@@ -9,6 +9,7 @@ Class {
 
 { #category : #accessing }
 ClapMismatch >> at: aSpec [
+self flag: 'identifierOrSpec?'.
```

Comment:
indentation?

Reply:
…it's a flag 😒

More seriously the problem here is that I need to check that the parameter makes sense wrt. the receiver's specification; otherwise it's a programmer error.

Reply:
I'm not criticising the code, I'm saying it should be properly formatted, at least indented, that's all :)